stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/ChannelUserRead.kt

```diff
@@ -5,5 +5,6 @@ import java.util.Date
 public data class ChannelUserRead(
```

Comment:
Let's add KDocs here

Reply:
Instead of using "normal comments" over the class property, we should use the proper KDoc Sintax:
https://kotlinlang.org/docs/kotlin-doc.html

For this example, we should have something like:
```
/**
* Info about how many unread messages [...]
*
* @property user The user which has read [...]
* @property lastRead The time of the last read message.
* [...]
**/
```